docs/src/modules/getting-started/pages/starthere.adoc

```diff
@@ -0,0 +1,36 @@
+= Getting started
+
+include::ROOT:partial$include.adoc[]
+
+You might be excited to start your journey learning and building with Akka, but there are a few things you need to get set up first. This will only take a minute.
+
+== Prerequisites
+
+include::ROOT:partial$local-dev-prerequisites.adoc[]
+* https://doc.akka.io/operations/cli/installation.html[Akka CLI]
+* Git
+* AI provider API keys (for AI samples)
+* https://account.akka.io/token[Development token for Maven artifact downloads]
+
+== Generating a secure repository token
+
+When you go to the https://account.akka.io/token[repository token] page, you will be presented with two options:
+
+* **Register** - create a new support account.
+* **Current customer** - select this if you have already created a support account.
+
+Remember that this account is an _Akka **support** account_ and is not the same thing as an Akka cloud account. This account is specifically for interacting with support and downloading Akka dependency binaries from the secure Maven repository.
```

Comment:
support is often associated with a cost, shall we mention that it's free to create a development download token?